pypeit/tests/test_history.py

```diff
@@ -95,6 +95,7 @@ def test_add_reduce():
     spectrograph_def_par = spectrograph.default_pypeit_par()
     ftype_bitmask = FrameTypeBitMask()
 
+    # TODO: Does this need to use np.int_?  Can we just use 'int'?
```

Comment:
The bitmask.py code calls "astype" on the argument, which doesn't work on int but does on np.int_.

Reply:
Hmmm.... Okay.  Leaving it for now.  I don't remember why these type casting calls are there.